core/src/main/java/org/mule/config/builders/DefaultsConfigurationBuilder.java

```diff
@@ -106,6 +106,7 @@ protected void doConfigure(MuleContext muleContext) throws Exception
         registry.registerObject(MuleProperties.OBJECT_EXPRESSION_LANGUAGE, new MVELExpressionLanguageWrapper(muleContext));
         registry.registerObject(MuleProperties.OBJECT_CONNECTOR_MESSAGE_PROCESSOR_LOCATOR, new MuleConnectorOperationLocator());
         registry.registerObject(MuleProperties.OBJECT_TIME_SUPPLIER, new TimeSupplier());
+        registry.registerObject(MuleProperties.OBJECT_MULE_CONTEXT, muleContext);
```

Comment:
it is there for the SpringRegistry, I'm adding it here as well so that it's available for injection on the test cases